.travis.yml

```diff
@@ -204,4 +212,5 @@ matrix:
           then
               STATUSM="$STATUSM ($(python -c "print '%+.2f' % (100*($CURR-$PREV)/$PREV.0)")%)"
           fi
+        - coveralls --gcov-options '\-lp'
```

Comment:
nit: I would move this up to immediately after the "after_success" label. The code blob above and the bash command below are both a part of the same command to update the github status.